internal/app/app_test.go

```diff
@@ -81,6 +81,32 @@ func TestLoadCliConfig_with_validation(t *testing.T) {
 	}
 }
 
+func TestLoadCliConfig_defaults(t *testing.T) {
+	cliConfig := &CliConfig{
+		CredentialsFile: "./test-fixtures/test_credentials.json",
+		ClusterName:     "test",
+		ClusterLocation: "europe-central2",
+		ProjectName:     "my-project",
+	}
+	pa := PolicyAutomationApp{ctx: context.Background()}
+	err := pa.LoadCliConfig(cliConfig, nil)
+	if err != nil {
+		t.Fatalf("got error; expected nil")
```

Comment:
error message could be more descriptive, and maybe contain the error message?

Reply:
Good point!